listenbrainz/tests/integration/test_playlist_api.py

```diff
@@ -335,7 +335,9 @@ def test_playlist_create_with_collaborators(self, mock_fetch_playlist_recording_
         """ Test to ensure creating a playlist with collaborators works """
 
         playlist = get_test_data()
-        playlist["playlist"]["extension"][PLAYLIST_EXTENSION_URI]["collaborators"] = [self.user2["musicbrainz_id"]]
+        # Owner is in collaborators, should be filtered out
```

Comment:
is this a TODO, or a description of this test? If it's a description, can you change it to "If the owner is in collaborators, it should be filtered out"